### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by adamrturman**
## What Changes

- Updated docs/readme.md to include recent version numbers, dates, links, etc.
- Added docs/validator_guide.md (which is a update of the steps included in the "Validating Kava-4 Mainnet" [article](https://medium.com/kava-labs/validating-kava-mainnet-72fa1b6ea579)
- Added upgrade guides for validators, wallets, and exchanges
- Added link to these guides in the main readme.

### CODE REVIEW

File: docs/upgrade_oracle.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# kava-8 Oracle Update Guide
+The Kava blockchain is upgrading and minor updates are needed for oracles
+
+## Update Guide For Standalone Oracle Operators
+1. Install the latest version of kava-tools
+2. In your `.env` configuration file, add an entry for `FEE`. This will set the default fee (in ukava) for each oracle transaction (can be 0).
+   - `FEE=”10000”`
+3. In your `.env` configuration file, update the MARKET_IDS entry to include the latest markets for kava-5:
+   - `MARKET_IDS="bnb:usd,bnb:usd:30,btc:usd,btc:usd:30,xrp:usd,xrp:usd:30,busd:usd,busd:usd:30,kava:usd,kava:usd:30,hard:usd,hard:usd:30,usdx:usd"`
```
Comment (karzak) [inline docs/upgrade_oracle.md:9]:
```suggestion
   - `MARKET_IDS="bnb:usd,bnb:usd:30,btc:usd,btc:usd:30,xrp:usd,xrp:usd:30,busd:usd,busd:usd:30,kava:usd,kava:usd:30,hard:usd,hard:usd:30,usdx:usd,usdx:usd:30"`
```

File: docs/upgrade_oracle.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+   - btc:usd:30
+   - xrp:usd
+   - xrp:usd:30
+   - busd:usd
+   - busd:usd:30
+   - kava:usd
+   - kava:usd:30
+   - hard:usd
+   - hard:usd:30
+   - usdx:usd
```
Comment (karzak) [inline docs/upgrade_oracle.md:34]:
```suggestion
   - usdx:usd 
   - usdx:usd:30
```

File: docs/upgrade_oracle.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# kava-8 Oracle Update Guide
```
Comment (karzak) [inline docs/upgrade_oracle.md:1]: Put these files (upgrade_oracle, validator, and wallet) inside a directory for `kava-8`. So, `docs/kava-8/update_oracle.md`, etc.

File: docs/validator_guide.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
+# Validator Guide
```
Comment (karzak) [inline docs/validator_guide.md:1]: This is fine to keep at the top level `docs` directory 👍

File: docs/validator_guide.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+export GOPATH=$HOME/go
+export GO111MODULE=on
+export PATH=$PATH:/usr/local/go/bin:$HOME/go/bin
+EOF
+source ~/.profile
+```
+To verify that Go is installed:
+```bash
+go version
+# Should return go version go1.15.14 linux/amd64
```
Comment (karzak) [inline docs/validator_guide.md:44]:
```suggestion
# Should return go version go1.17.1 linux/amd64
```

**Review by karzak — CHANGES_REQUESTED** [review]
Few nits, otherwise looks good

File: docs/upgrade_oracle.md (hunk 3 above)
Comment (adamrturman) [reply docs/upgrade_oracle.md]: Ok - addressed [here](https://github.com/Kava-Labs/kava/pull/1020/commits/ec60bf16c36544df57e84d4b30585f410b471e13)

**Review by karzak — APPROVED** [review]

**Pull request merged**